Python/Tests/Core/EnvironmentListTests.cs

```diff
@@ -771,6 +772,121 @@ public class EnvironmentListTests {
             TestFileNameEllipsis("", "", "", "");
         }
 
+        [TestMethod, Priority(0)]
+        public void FilterPython2Interpreter() {
```

Comment:
We try to keep the version at the very end of the test class name / method name.

Since the method you are actually testing is "ExcludeInterpreter", I'd probably name them "ExcludeInterpreterPython2", "ExcludeInterpreterPython3", "ExcludeInterpreterConda2", etc.